### DISCUSSION TITLE
Mac crashed, now GUI doesn't find the (unfinished) Plot anymore

### DISCUSSION THREAD

**Discussion by thejulianw** [Chia-Network/chia-blockchain#1451] [Q&A]
Hi,

Super new to this whole thing.
I created a plot on an external hard drive (both primary and final destination on HDD) and had it run for a couple hours.

My Mac crashed over night and now the Chia GUI acts as though there is no Plot, all I can do is start a new one.
On the external HDD, the files are still showing, I just can't get the GUI to pick it up again.

Tried restarting everything and reentering the Plot directory.

Anything I can do? Do I have to start it all over? When I create a new Plot, will it find the existing tables or are they useless now?

Cheers,

Julian

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
